.circleci/config.yml

```diff
@@ -161,7 +161,7 @@ run_oss_benchmark: &run_oss_benchmark
       name: Run OSS Benchmark
       command: |
         python benchmarks/oss.py --world_size 4 --epochs 2
-        python benchmarks/oss.py --check_regression --world_size 4 --optim_type oss_sharded_ddp
+        python benchmarks/oss.py --check_regression --world_size 4 --optim_type oss_sharded_ddp --epochs 12
```

Comment:
added this so that we run 12 instead of 10 epochs to reduce the variability. runtime is not significantly longer.